pyproject.toml

```diff
@@ -16,3 +16,14 @@ source = ["scripts", "app"]
 
 [tool.coverage.report]
 show_missing = true
+
+[tool.mypy]
+check_untyped_defs = true
+python_version = 3.9
+
+[[tool.mypy.overrides]]
+module = [
+    "msal.*",
```

Comment:
msal doesn't have types :( There's a feature request for it.

Reply:
Bummer!